lib/zclient.c

```diff
@@ -2679,6 +2679,19 @@ int zapi_labels_decode(struct stream *s, struct zapi_labels *zl)
 	}
 
 	STREAM_GETW(s, zl->nexthop_num);
+
+	if (zl->nexthop_num > MULTIPATH_NUM) {
+		char buff[PREFIX2STR_BUFFER];
+
+		prefix2str(&zl->route.prefix, buff, sizeof(buff));
+		flog_warn(
+			EC_LIB_ZAPI_ENCODE,
+			"%s: Prefix %s has %d nexthops, but we can only use the first %d",
+			__func__, buff, zl->nexthop_num, MULTIPATH_NUM);
```

Comment:
prefix output - don't we have a whiz-bang format-specifier available now?

Reply:
Hmm